python/pdstools/reports/ModelReport.qmd

```diff
@@ -103,22 +103,23 @@ kwargs = dict()
 ```{python}
 # Predictor data for one model ID
```

Comment:
@yusufuyanik1  perhaps we need to fix something here to support the call from streamlit, pls have a look